app/views/koi/admin_crud/_form_field_inline.html.erb

```diff
@@ -70,7 +70,6 @@
         $(this).parent("h3").toggleClass("nested-fields-visible");
         $(this).parent("h3").parent(".nested-fields").siblings().find("h3").removeClass("nested-fields-visible").next(".nested-inline-fields").hide();
         $(this).closest(".nested-fields").children(".nested-inline-fields").toggle();
-        reInitCkEditor();
```

Comment:
It was fixing the floating toolbar being the wrong size. 
I went through this with you the other day but it went over your head at the time 😛 

* When the accordion closes, the editor is now technically 0px wide and 0px tall. The accordions are closed by default. All editors are now 0x0 by default. 
* The floating toolbar needs to know where to sit inside the editor, if the editor is zero by zero, the toolbar becomes zero pixels wide too. Technically this is correct behaviour because the toolbar should always be as wide as the editor. 
* When the accordion is opened back up, the CKEditor gets it's true size because it is just a textarea that has physicality on the page. It's not floated, it's not positioned, it's a static, block-level element so it reflows like any good element would. The toolbar is positioned absolutely to the editor so it doesn't know that it needs to be resized. You saw an example of this with the YourSAy test on staging, where the floating toolbar wasn't resizing to the editor. 
* The re-initialisation function here removed all CKEditor bindings from the textarea and then re-applied them. The toolbar then gets generated while the editor is the correct size, so the toolbar is now the right size. Because it was such a volatile solution and removing all the CKEditor stuff, it was losing any previous content it had in the editor. 

This is the same for all CKEditor re-initialisations. It was only ever to fix the floating toolbar for previously-hidden editors. The only time a CKEditor should be initialised is if it's newly inserted in to the page after CKEditors have already been initialised (eg. added via cocoon, ajax, lightbox etc.)

The updates to the `wysiwyg()` function also take this in to account by first checking if the editor has already been initialised by checking if it's in `CKEditor.instances` before trying to initialise it. So feel free to call `$(".wysiwyg").wysiwyg()` everywhere and anywhere.